RULES.md

```diff
@@ -702,6 +702,39 @@ A collection of community rules that are not currently included in any of the pr
 | LambdaFunctionUrlAuth                 | The Lambda Function URL allows for public, unauthenticated access.                                               | AWS Lambda Function URLs allow you to invoke your function via a HTTPS end-point, setting the authentication to NONE allows anyone on the internet to invoke your function.                                                                                                                                                          |
 | LambdaEventSourceSQSVisibilityTimeout | The SQS queue visibility timeout of Lambda Event Source Mapping is less than 6 times timeout of Lambda function. | Setting the visibility timeout to [at least 6 times the Lambda function timeout](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-configure-lambda-function-trigger.html) helps prevent configurations resulting in duplicate processing of queue items when the Lambda function execution is retried. |
 
+
```

Comment:
1. Update the `Cause`s to match with the feedback I gave for the rule descriptions
2. This section should come before the `Additional Rules` section